packages/backend-proxy-middleware/src/base/proxy.ts

```diff
@@ -38,7 +38,7 @@ export const ProxyEventHandlers = {
      * @param _options (not used)
      */
     onProxyReq(proxyReq: ClientRequest, _req?: IncomingMessage, _res?: ServerResponse, _options?: ServerOptions) {
-        if (proxyReq.path.indexOf('Fiorilaunchpad.html') !== -1 && !proxyReq.headersSent) {
+        if (proxyReq.path && proxyReq.path.indexOf('Fiorilaunchpad.html') !== -1 && !proxyReq.headersSent) {
```

Comment:
Can't you merge it to `proxyReq.path?.indexOf('Fiorilaunchpad.html') !== -1

Reply:
Nope, `proxyReq.path?.indexOf('Fiorilaunchpad.html') !== -1` will evaluate to `true` and this will not be correct for this scenario.

Reply:
Good point, `proxyReq.path?.includes('Fiorilaunchpad.html')` should do the trick.